flowdock-text.js

```diff
@@ -963,25 +963,29 @@ if (typeof FlowdockText === "undefined" || FlowdockText === null) {
     };
   };
 
-  FlowdockText.mentionsAllTags = ["@everyone", "@everybody", "@all", "@anyone", "@anybody"];
```

Comment:
In that case, we could just remove the alltogether. I don't see any point in going half way.